src/components/deck/deck.scss

```diff
@@ -0,0 +1,7 @@
+@import "../../frameless";
+
+@include responsive-layout (".deck", ".slide");
+
+.deck {
+    min-height: 100vh;
```

Comment:
Does our webpack config handle fallbacks for this when not supported?
http://caniuse.com/#feat=viewport-units